extensions/agroal/deployment/src/main/java/io/quarkus/agroal/deployment/AgroalProcessor.java

```diff
@@ -205,6 +214,10 @@ void generateDataSourceSupportBean(AgroalRecorder recorder,
             BuildProducer<UnremovableBeanBuildItem> unremovableBeans) {
         additionalBeans.produce(new AdditionalBeanBuildItem(JdbcDriver.class));
 
+        if (capabilities.isPresent(Capability.OPENTELEMETRY_TRACER)) {
+            recorder.useOpenTelemetryForTracing();
+        }
```

Comment:
I'm not sure I understand why you need that? And why you didn't follow the pattern we used for OpenTracing. I'm probably missing something but I would like to understand.

Reply:
Let's go with `.telemetry` and have a different setting. Then you can be consistent and use the same pattern.

If one day, we remove OpenTracing support, it will also be easier.